redshift_connector/cursor.py

```diff
@@ -6,7 +6,8 @@
 from itertools import count, islice
 from typing import TYPE_CHECKING
 from warnings import warn
-
+from psycopg2 import extras
+import csv
```

Comment:
Could this import be moved to the `insert_data_bulk()` method as it's only used in this method.